storage/src/test/scala/ch/epfl/bluebrain/nexus/storage/DiskStorageSpec.scala

```diff
@@ -317,6 +330,84 @@ class DiskStorageSpec
       }
     }
 
+    "copying" should {
```

Comment:
We could complete this by setting the original as read-only and check that that those attributes are preserved

Reply:
Done in https://github.com/BlueBrain/nexus/pull/4511/commits/b774a6e488fcfb1f7f709a71775f158ad1b02668